index.html

```diff
@@ -575,7 +575,8 @@ <h4>
       <ul>
         <li>
           A single mandatory <a>URI record</a> that refers to the
-          <a>smart poster</a> content.
+          <a>smart poster</a> content. A <a>smart poster</a> overrides any
+          other <a>URI records</a> in an <a>NDEF message</a>.
```

Comment:
according to NFC spec link? or somthing like that

Reply:
The Smart Poster RTD, obviously. The whole section is about that one and it's stated/linked in the previous sentence. I don't expect to link it in every sentence in that section?

Reply:
Spec says 

> If an NDEF message contains one or multiple URI records (see [RTD_URI]) in addition to the
Smart Poster record at the top level (i.e., not nested), the Smart Poster record overrides the other
URI records. The NDEF application SHALL use only the Smart Poster record.

However, I'm not sure we should actually add this bit to the spec, as it is up to the app to actually consider a second URI record.

Reply:
Saying it another way, I don't think Web NFC should delete/remove URI records from NDEF messages with smart poster.

Reply:
I thought Web NFC as an application from the NFC Forum point of view. As I see it,  an app doesn't really have the right to consider a second URI record in a smart poster, unless it wants to me non-conformant to the NFC spec.